editor/scene_tree_dock.cpp

```diff
@@ -617,7 +619,7 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 		} break;
 		case TOOL_CUT:
 		case TOOL_COPY: {
-			if (!edited_scene || (p_tool == TOOL_CUT && !_validate_no_foreign())) {
+			if (!edited_scene || !_validate_no_foreign()) {
```

Comment:
Why this change? It makes it impossible to copy foreign nodes. If you are worried about copying exposed nodes, you can disable exposing when node is put into clipboard.